Source/NSCell.m

```diff
@@ -2966,31 +2979,70 @@ - (NSSize) _sizeText: (NSString*)title
 - (NSAttributedString*) _drawAttributedString
 {
   if (!_cell.is_disabled)
-    {  
+    {
       return [self attributedStringValue];
     }
   else
     {
       NSAttributedString *attrStr = [self attributedStringValue];
       NSDictionary *attribs;
       NSMutableDictionary *newAttribs;
-              
-      attribs = [attrStr attributesAtIndex: 0 
+
+      attribs = [attrStr attributesAtIndex: 0
                          effectiveRange: NULL];
       newAttribs = [NSMutableDictionary dictionaryWithDictionary: attribs];
       [newAttribs setObject: [NSColor disabledControlTextColor]
                   forKey: NSForegroundColorAttributeName];
-      
+
       return AUTORELEASE([[NSAttributedString alloc]
                              initWithString: [attrStr string]
                              attributes: newAttribs]);
     }
 }
 
+
```

Comment:
A huge -- but necessary -- amount of cleanup obscures the fact that this is the new code. Not sure how I feel about this. I can't see if anything else changed in this file.

Reply:
The whitespace changes are the result of one careless use of delete-trailing-whitespace. There is an option on Github to ignore such changes.

Reply:
Yes, I didn't look at PR description as I was asked in a chat to take a look at this. Sorry about that!

Either way, it took me quite a bit of time to sift through it, and I'm not sure my comments are useful, but eh.